app/src/main/kotlin/com/intive/picover/profile/view/OptionsList.kt

```diff
@@ -0,0 +1,59 @@
+package com.intive.picover.profile.view
+
+import android.content.Intent
+import android.net.Uri
+import androidx.browser.customtabs.CustomTabsIntent
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Icon
+import androidx.compose.material3.NavigationDrawerItem
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.google.android.gms.oss.licenses.OssLicensesMenuActivity
+import com.intive.picover.profile.model.OptionsListItem
+
+@Composable
+fun OptionsList(onLogoutClick: () -> Unit, onDeleteAccountCLick: () -> Unit) {
+	val context = LocalContext.current
+	val items = listOf(
+		OptionsListItem.Licenses(
+			onClick = {
+				context.startActivity(Intent(context, OssLicensesMenuActivity::class.java))
+			},
+		),
+		OptionsListItem.Logout(onClick = onLogoutClick),
+		OptionsListItem.DeleteAccount(onClick = onDeleteAccountCLick),
+		OptionsListItem.GitHub(
+			onClick = {
+				CustomTabsIntent
+					.Builder()
+					.build()
+					.launchUrl(context, Urls.github)
+			},
+		),
+	)
+	LazyColumn {
+		items(items) {
+			NavigationDrawerItem(
+				icon = {
+					Icon(
+						painter = painterResource(id = it.iconId),
+						contentDescription = stringResource(id = it.textId),
+					)
+				},
+				label = {
+					Text(text = stringResource(id = it.textId))
+				},
+				onClick = it.onClick,
+				selected = false,
+			)
+		}
+	}
+}
+
+private object Urls {
+	val github: Uri = Uri.parse("https://github.com/intive/Picover")
```

Comment:
1. `Urls` should be singular
2. Redundant type
3. "https://github.com/intive/Picover" should be placed in resources

Reply:
2. "Declaration has type inferred from a platform call, which can lead to unchecked nullability issues. Specify type explicitly as nullable or non-nullable."
3. Do you mean non-translatable string resource?

Reply:
2. Ok
3. Yes, it could be a separate resource file, to not mix it with `strings.xml`

Reply:
Done